### PR TITLE
add multicluster global hub resources.

### PR DESCRIPTION
**Pull request by morvencao**
**Related Issue:**  https://issues.redhat.com/browse/ACM-4574

**Description of Changes:**

Add Multicluster Global Hub resources for must gather scripts.

**What resource is being added**:
MulticlusterGlobalHub
Kafka
KafkaTopic
PostgresCluster

Also 
added namespaces of kafka-operator and postgres-operator in Hub cluster.
added namespace of `open-cluster-management-global-hub-system` namespace in managed cluster.

**Is this a Hub or Managed cluster change?:**
Hub Cluster  and Managed Cluster

**Notes:**

### CODE REVIEW

File: collection-scripts/gather [hunk 1]
```diff
@@ -268,6 +272,25 @@ gather_hub(){
     oc adm inspect submarinerconfigs.submarineraddon.open-cluster-management.io --all-namespaces --dest-dir=must-gather
     oc adm inspect brokers.submariner.io --all-namespaces --dest-dir=must-gather
 
+    # Multicluster GlobalHub information
+    oc adm inspect multiclusterglobalhubs.operator.open-cluster-management.io --all-namespaces --dest-dir=must-gather
```
Comment (jlpadilla) [inline collection-scripts/gather:276]: Can we ad a conditional to only collect kafka and postgres data when global hub is enabled?
```bash
oc adm inspect multiclusterglobalhubs.operator.open-cluster-management.io --all-namespaces --dest-dir=must-gather
if [ $? = 0 ]
then
  # Kafka
  # Postgresql
fi
```

File: collection-scripts/gather [hunk 2]
```diff
@@ -268,6 +272,25 @@ gather_hub(){
...
+    # kafka
+    oc adm inspect kafkas.kafka.strimzi.io --all-namespaces --dest-dir=must-gather
+    oc adm inspect kafkatopics.kafka.strimzi.io --all-namespaces --dest-dir=must-gather
+    oc adm inspect kafkausers.kafka.strimzi.io --all-namespaces --dest-dir=must-gather
+    KAFKANS=`oc get kafkas.kafka.strimzi.io --all-namespaces --no-headers=true -o custom-columns=NAMESPACE:.metadata.namespace | sort -u`
+    for kns in $KAFKANS;
+    do
+        oc adm inspect ns/"$kns"  --dest-dir=must-gather
+    done
```
Comment (bjoydeep) [inline collection-scripts/gather:285]: when we try to gather data from ns kafka, we get an error. Not sure if you already know why/this @morvencao :
```
Gathering data for ns/kafka...
E0510 11:02:52.765512   82553 portforward.go:400] an error occurred forwarding 37587 -> 2888: error forwarding port 2888 to pod b2f2e0c74f8b2369693a800f2f2040331c7f28820903dbace3f2c6c9b6d7733f, uid : port forward into network namespace "/var/run/netns/58f163ac-2bcd-46d0-a875-137708a3d72f": failed to connect to localhost:2888 inside namespace b2f2e0c74f8b2369693a800f2f2040331c7f28820903dbace3f2c6c9b6d7733f: dial tcp [::1]:2888: connect: connection refused
```

Comment (bjoydeep) [reply collection-scripts/gather:285]: Should we also be adding:
which is the namespace under which the gh-agent runs on the local hub

The other thing that we need to capture is some data about the postgreSQL tables itself. Any thoughts on how to get that? We should dump the contents of local_status.compliance_history_job_log table in a CSV..
or better still trigger some postgres utility to get some data

File: collection-scripts/gather (hunk 1 above)
Comment (morvencao) [reply collection-scripts/gather:276]: nice catch! done.

File: collection-scripts/gather (hunk 2 above)
Comment (morvencao) [reply collection-scripts/gather]: Currently must-gather mainly gather kubernetes resources and service logs. For data in database, not sure if we should gather with this. We may need extra tool to let customer dump database tables.

For the `error forwarding port ..`, looks like there is similar issue logged at https://bugzilla.redhat.com/show_bug.cgi?id=1857597

From the issue, the `error forwarding port ..` doesn't affect the must-gather function @bjoydeep 

> Although there is portforward error, it does not affect the function, move to VERIFIED, feel free to reopen it if you think it is a bug

**Pull request merged**